markhor_flippers/src/markhor_hw_interface_flippers.cpp

```diff
@@ -213,105 +232,115 @@ void MarkhorHWInterfaceFlippers::write()
-    ROS_INFO_THROTTLE(1,"rear_left_drive_base_position : %f", rear_left_drive_base_position_);
-    ROS_INFO_THROTTLE(1,"accumulator_rl : %f", accumulator_rl_);
-    ROS_INFO_THROTTLE(1,"Target : %f", rear_left_drive_base_position_ + accumulator_rl_ + joint_position_command_[2]);
-    ROS_INFO_THROTTLE(1,"Drive %d : output voltage : %f", rear_left_drive_->GetDeviceID(),
-             rear_left_drive_->GetMotorOutputVoltage());
+    ROS_INFO_THROTTLE(1, "position RL command : %f", joint_position_command_[2]);
+    ROS_INFO_THROTTLE(1, "RL lower limit : %f", rear_left_drive_lower_limit_);
+    ROS_INFO_THROTTLE(1, "RL upper limit : %f", rear_left_drive_upper_limit_);
+    ROS_INFO_THROTTLE(1, "rear_left_drive_base_position : %f", rear_left_drive_base_position_);
+    ROS_INFO_THROTTLE(1, "accumulator_rl : %f", accumulator_rl_);
+    ROS_INFO_THROTTLE(1, "Target : %f", target_rl_);
+    ROS_INFO_THROTTLE(1, "Drive %d : output voltage : %f", rear_left_drive_->GetDeviceID(),
+                      rear_left_drive_->GetMotorOutputVoltage());
   }
   else if (drive->GetDeviceID() == drive_rr_id_)
   {
-    ROS_INFO_THROTTLE(1,"position RR command : %f", joint_position_command_[3]);
-    ROS_INFO_THROTTLE(1,"RR lower limit : %f", rear_right_drive_lower_limit_);
-    ROS_INFO_THROTTLE(1,"RR upper limit : %f", rear_right_drive_upper_limit_);
-    ROS_INFO_THROTTLE(1,"rear_right_drive_base_position : %f", rear_right_drive_base_position_);
-    ROS_INFO_THROTTLE(1,"accumulator_rr : %f", accumulator_rl_);
-    ROS_INFO_THROTTLE(1,"Target : %f", rear_right_drive_base_position_ + accumulator_rr_ + joint_position_command_[3]);
-    ROS_INFO_THROTTLE(1,"Drive %d : output voltage : %f", rear_right_drive_->GetDeviceID(),
-             rear_right_drive_->GetMotorOutputVoltage());
+    ROS_INFO_THROTTLE(1, "position RR command : %f", joint_position_command_[3]);
+    ROS_INFO_THROTTLE(1, "RR lower limit : %f", rear_right_drive_lower_limit_);
+    ROS_INFO_THROTTLE(1, "RR upper limit : %f", rear_right_drive_upper_limit_);
+    ROS_INFO_THROTTLE(1, "rear_right_drive_base_position : %f", rear_right_drive_base_position_);
+    ROS_INFO_THROTTLE(1, "accumulator_rr : %f", accumulator_rl_);
+    ROS_INFO_THROTTLE(1, "Target : %f", target_rr_);
+    ROS_INFO_THROTTLE(1, "Drive %d : output voltage : %f", rear_right_drive_->GetDeviceID(),
+                      rear_right_drive_->GetMotorOutputVoltage());
   }
   else
   {
-    ROS_INFO_THROTTLE(1,"Device ID is not found");
+    ROS_INFO_THROTTLE(1, "Device ID is not found");
     return;
   }
 
-  ROS_INFO_THROTTLE(1,"GetStatorCurrent %f", drive->GetStatorCurrent());
-  ROS_INFO_THROTTLE(1,"GetPulseWidthPosition %d", drive->GetSensorCollection().GetPulseWidthPosition());
-  ROS_INFO_THROTTLE(1,"GetClosedLoopError %d", drive->GetClosedLoopError(0));
-  ROS_INFO_THROTTLE(1,"GetClosedLoopTarget %f", drive->GetClosedLoopTarget(0));
+  ROS_INFO_THROTTLE(1, "GetStatorCurrent %f", drive->GetStatorCurrent());
```

Comment:
REF: Line 281, we are publishing OutputCurrent, shouldn't we use the same value for the logging? this may cause confusion in the future, getStatorCurrent might be a falcon specific function as in our case, the current should be flowing in the rotor as we are using dc motors

Reply:
Edit, this is not your code, might be worth removing to make things simpler